make/cmake_format.py

```diff
@@ -0,0 +1,137 @@
+# How wide to allow formatted cmake files
+line_width = 80
```

Comment:
It seems like there are lines which are longer than 80 characters?

Reply:
cmake_format.py isn't perfect and CMake lacks good line continuation options in some cases.  I'm actually wary of further use of cmake_format.

Reply:
I'm strongly in support of having files get auto formatted.

I'd prefer *all* files be auto formatted if possible.